### PR TITLE
Home: Improve text and readability

### PR DESCRIPTION
**Pull request by grahamperrin**
skunkworks: one word, not two. 

Whilst here, some other suggested changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
...
-                <li>Someone (YOU!) are willing to drive forward</li>
+                <li>Can begin with a prototype or <a href="https://en.wikipedia.org/wiki/Minimum_viable_product">MVP</a>, then scale from there</li>
+                <li>Can be achieved with a small group (2–4) of diversely-skilled people</li>
+                <li>Are not just related to <em>code</em> (src, ports, doc)</li>
+                <li>Are not just feature requests</li>
+                <li>Someone (YOU!) is willing to drive forward</li>
             </ul>
             <p>
-                Think <a href="https://en.wikipedia.org/wiki/Skunkworks_project" target="_blank">“Skunk Works”</a> for FreeBSD. Welcome to The Lab!
+                Think <a href="https://en.wikipedia.org/wiki/Skunkworks_project" target="_blank"><em>skunkworks</em></a> for FreeBSD. Welcome to the Labs!
```
Comment (koobs) [inline index.html]: * Leave Skunkworks capitalized
* Leave Lab un-pluralized. I realise the eponymous label is 'bsdlab**s**', but the 'labs' is more 'multiple projects' than 'multiple laboratories'. Maybe I'm wrong.

File: index.html [hunk 2]
```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
...
             </p>
 
             <h2>Getting started</h2>
             <ul>
                 <li>Join us on <a href="https://github.com/bsdlabs" target="_blank">GitHub</a></li>
-                <li>Stop by #bsdlabs @ Libera Chat or our Discord (jump on IRC for an invite)</li>
+                <li>Stop by #bsdlabs @ Libera Chat or our Discord (for an invite: jump on IRC, or into the <a href="https://discord.com/channels/727023752348434432/757543661058654269">#community</a> channel in Discord for FreeBSD)</li>
```
Comment (koobs) [inline index.html]: * Link Discord to `https://discord.gg/9vZzzxz7y9` directly (i've done this elsewhere)
* Remove freebsd discord (#community) link, not relevent (globally, yet)
* Re-order Discord before IRC

File: index.html [hunk 3]
```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
...
             </ul>
 
             <h2>Useful links</h2>
             <ul>
-                <li><a href="https://freebsd.org" target="_blank">FreeBSD.org</a></li>
-                <li><a href="https://bugs.freebsd.org" target="_blank">FreeBSD Issue Tracker</a></li>
+                <li><a href="https://www.freebsd.org/" target="_blank">The FreeBSD Project</a></li>
+                <li><a href="https://bugs.freebsd.org/" target="_blank">FreeBSD Bugzilla</a>  (issue tracker)</li>
```
Comment (koobs) [inline index.html]: * Rename Useful to Other (or better term)
* Remove these links (www/bugs, not relevent). We'll link off to our playbook and/or other bsdlabs major repos/docs when they're public

**Review by koobs — CHANGES_REQUESTED** [review]
Thanks for this Graham

**Comment by grahamperrin** [comment]
@koobs feel free to edit the file directly :-)

**Comment by koobs** [comment]
> @koobs feel free to edit the file directly :-)

I don't unless asked first, some peeps like to finish up themselves. I can make post-PR edits after merge.

Thank you again for the review and updates

**Pull request merged**